ownership-chain/runtime/src/lib.rs

```diff
@@ -334,7 +331,14 @@ impl pallet_authorship::Config for Runtime {
 }
 
 parameter_types! {
-	pub const ExistentialDeposit: Balance = EXISTENTIAL_DEPOSIT;
+/// The minimum amount required to keep an account open, set to zero in this case.
```

Comment:
is it possible to indent this to match with `pub const` line? or does it look unindented like this only to me?
![Screenshot 2023-11-13 at 16 50 38](https://github.com/freeverseio/laos/assets/137785454/1601e69d-7302-4627-830a-2a43d3f74118)